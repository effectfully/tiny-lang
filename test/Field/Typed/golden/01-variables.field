let ?bool    = ?i;
let field'   = i;
let #vector  = #i;