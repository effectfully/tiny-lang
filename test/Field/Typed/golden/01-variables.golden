ext ef_0 : field;
ext eb_1 : bool;
ext ev_2 : vector;
let b_3 : bool = (eb_1 : bool);
let f_4 : field = (ef_0 : field);
let v_5 : vector = (ev_2 : vector);
